File: fuelnet_back/src/main/java/com/fuelnet/fuelnet/dto/PaymentResponse.java
package com.fuelnet.fuelnet.dto;

import com.fuelnet.fuelnet.enums.PaymentStatus;
import lombok.Builder;
import lombok.Data;
import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@Builder
public class PaymentResponse {
    private Long id;
    private String clientName;
    private String clientEmail;
    private String fuelType;
    private BigDecimal gallons;
    private BigDecimal amount;
    private PaymentStatus status;
    private LocalDateTime createdAt;
    private LocalDateTime paidAt;
}

File: FuelControl/app/src/main/java/co/edu/unipiloto/fuelcontrol/api/requests/PaymentResponse.kt
package co.edu.unipiloto.fuelcontrol.api.requests

data class PaymentResponse(
    val id: Long,
    val clientName: String,
    val clientEmail: String,
    val fuelType: String,
    val gallons: Double,
    val amount: Double,
    val status: String,
    val createdAt: String,
    val paidAt: String?
)